tests/vm.rs

```diff
@@ -557,10 +559,14 @@ fn run_expr_int() {
 
     let text = r#"io.run_expr "123" "#;
     let mut vm = make_vm();
-    let (result, _) = Compiler::new().run_io_expr::<String>(&mut vm, "<top>", text).unwrap();
-    let expected = "123 : Int";
-
-    assert_eq!(result, expected);
+    let (result, _) = Compiler::new().run_io_expr::<IO<String>>(&mut vm, "<top>", text).unwrap();
```

Comment:
Should make this clearer, it can still run non `IO` expressions, but if it is `IO` then the action is run as well. Might be a better API than this though